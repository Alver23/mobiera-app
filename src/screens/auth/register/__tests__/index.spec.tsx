// Dependencies
import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import { Text } from 'react-native-elements';

// Under test file
import RegisterScreen from '../index';

describe('<RegisterScreen />', () => {
  let component: ShallowWrapper;

  beforeEach(() => {
    component = shallow(<RegisterScreen />);
  });

  it('should save a snapshot of the component', () => {
    expect(component).toMatchSnapshot();
  });

  it('should render correctly', () => {
    expect(component.find(Text).exists()).toBeTruthy();
  });
});
