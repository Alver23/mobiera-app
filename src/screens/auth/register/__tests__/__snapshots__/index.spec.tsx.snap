// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RegisterScreen /> should save a snapshot of the component 1`] = `
<View>
  <Themed.Text>
    RegisterScreen
  </Themed.Text>
  <Themed.Button
    onPress={[Function]}
    title="Log in"
    type="outline"
  />
</View>
`;
